auditorium/views/main.js

```diff
@@ -96,30 +101,19 @@ function view (state, emit) {
       state.model.account.name
     )
     accountHeader = html`
-      <p class="dib pa2 br2 bg-black-05 mt0 mb2">
+      <p class="dib pa2 br2 ma0 mt3 ml3 ml0-ns mr3 mr0-ns bg-light-yellow">
         ${raw(copy)}
       </p>
     `
     pageTitle = state.model.account.name + ' | ' + state.title
   } else {
     pageTitle = __('user') + ' | ' + state.title
     accountHeader = html`
-      <p class="dib pa2 br2 bg-black-05 mt0 mb2">
-        ${raw(__('You are viewing your <strong>usage</strong> data.'))}
+      <p class="dib pa2 br2 ma0 mt3 ml3 ml0-ns mr3 mr0-ns bg-light-yellow">
+        ${raw(__('You are viewing your <strong>usage data.</strong> '))}
       </p>
-      ${state.model.allowsCookies ? null : html`<p class="dib pa2 black br2 bg-black-05 mt0 mb2">${__('Your browser does not allow 3rd party cookies. We respect this setting and collect only very basic data in this case, yet it also means we cannot display any data to you here.')}</p>`}
+      ${userAllowsCookies ? null : html`<p class="dib pa2 br2 ma0 mt3 ml3 ml0-ns mr3 mr0-ns bg-light-yellow">${__('Your browser does not allow 3rd party cookies. We respect this setting and collect only very basic data in this case, yet it also means we cannot display any data to you here.')}</p>`}
```

Comment:
Hmm, this is not really correct anymore as we currently drop anonymous events entirely. This is also why it's currently not even possible to access the Auditorium if a user has cookies disabled, thus this message will never be displayed. Should we remove it entirely?

Reply:
ok